src/frontend-pwa/vite.config.js

```diff
@@ -5,7 +5,7 @@ export default defineConfig({
   plugins: [
     VitePWA({
       registerType: "prompt",
-      includeAssets: ["favicon.ico", "apple-touch-icon.png", "android-chrome-192x192.png", "android-chrome-512x512.png", "logo-banner.svg", "gear-icon.svg"],
+      includeAssets: ["favicon.ico", "apple-touch-icon.png", "android-chrome-192x192.png", "android-chrome-512x512.png", "logo-banner.svg", "gear-icon.svg", "bc-logo-vertical.svg"],
```

Comment:
Can you not include the directory where the assets are located and a wildcard (*) instead of listing each asset individually?

Reply:
I can try that! I will make another ticket for that one though in case I break everything in a style update. Is that ok?

Reply:
We did have plans to refine the service worker functionality in the next sprint as well. This is currently modelled off of a tutorial to help us get started and has not been really touched since.